frame/collective/src/lib.rs

```diff
@@ -123,7 +129,10 @@ impl DefaultVote for MoreThanMajorityThenPrimeDefaultVote {
 
 pub trait Config<I: Instance=DefaultInstance>: frame_system::Config {
 	/// The outer origin type.
-	type Origin: From<RawOrigin<Self::AccountId, I>>;
+	type Origin: From<RawOrigin<Self::AccountId, I>>
+		+ From<frame_system::RawOrigin<Self::AccountId>>
+		+ From<<Self as frame_system::Config>::Origin>
+		+ Into<Result<RawOrigin<Self::AccountId, I>, <Self as Config<I>>::Origin>>;
```

Comment:
Where is this `Into<Result<_>>` actually implemented?

Reply:
In the runtime. This is used everywhere for the origin conversions.

Reply:
As a result of this PR, i def realized i have no idea all the conversions that the runtime generates for origins.

Reply:
Given our little frenzy here, would be good to make sure all of this is actually needed.